src/plugins/unified_search/public/filters_builder/filters_builder_utils.ts

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { DataViewField } from '@kbn/data-views-plugin/common';
+import { buildEmptyFilter, Filter } from '@kbn/es-query';
+import { cloneDeep } from 'lodash';
+import { ConditionTypes, getConditionalOperationType, isOrFilter } from '../utils';
+import type { FilterItem } from '../utils';
+import type { Operator } from '../filter_bar/filter_editor';
+
+const PATH_SEPARATOR = '.';
+
+/** to: @kbn/es-query **/
+const buildOrFilter = (filters: FilterItem) => {
+  const filter = buildEmptyFilter(false);
```

Comment:
can we create or filter type in kbn-es-query package, like we have it for example for exists filter:

https://github.com/elastic/kibana/blob/main/packages/kbn-es-query/src/filters/build_filters/exists_filter.ts